resources/views/ad-detail/default.twig

```diff
@@ -63,13 +63,15 @@
                         </div>
                         <span class="me-2">:</span>
                         <span class="fw-bold text-primary">
+                            {% if is_module_enabled('store', 'store') %}
```

Comment:
change to addBlock() function

Reply:
which module it suppose to put the code. it's already style :)